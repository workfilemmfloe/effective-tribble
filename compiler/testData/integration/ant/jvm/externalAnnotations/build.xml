<project name="Ant Task Test" default="build">
    <taskdef resource="org/jetbrains/kotlin/ant/antlib.xml" classpath="${kotlin.lib}/kotlin-ant.jar"/>

    <target name="build">
        <delete dir="${temp}/classes" failonerror="false"/>
        <mkdir dir="${temp}/classes"/>
        <javac destdir="${temp}/classes" includeAntRuntime="false" srcdir="${test.data}/root1">
            <withKotlin externalannotations="${test.data}/root1/b/">
                <externalannotations path="${test.data}/root1/a/"/>
            </withKotlin>
        </javac>
        <jar destfile="${temp}/hello.jar">
            <fileset dir="${temp}/classes"/>
        </jar>

        <java classname="hello.HelloPackage" fork="true">
            <classpath>
                <pathelement location="${temp}/hello.jar"/>
                <pathelement location="${kotlin.runtime.jar}"/>
            </classpath>
        </java>
    </target>
</project>
