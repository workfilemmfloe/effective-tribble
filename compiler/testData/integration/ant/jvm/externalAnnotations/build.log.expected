OUT:
Buildfile: [TestData]/build.xml

build:
    [mkdir] Created dir: [Temp]/classes
    [javac] Compiling 2 source files to [Temp]/classes
    [javac] Compiling [[TestData]/root1] => [[Temp]/classes]
    [javac] Running javac...
      [jar] Building jar: [Temp]/hello.jar
     [java] Hello, a!
     [java] Java Hello

BUILD SUCCESSFUL
Total time: [time]

Return code: 0
